nms/app/packages/magmalte/app/views/equipment/EnodebDetailSummaryStatus.js

```diff
@@ -74,6 +74,10 @@ export function EnodebStatus() {
         status: enbInfo.enb.enodeb_config?.managed_config?.transmit_enabled,
         tooltip: 'current transmit configuration on the eNodeB',
       },
+      {
+        category: 'Subscribers',
+        value: enbInfo.enb_state?.ues_connected ?? 0,
```

Comment:
Is there actually a valid situation currently when `ues_connected` would be undefined? I think perhaps we should modify the swagger definition for `enodeb_state` and here we can have `value` take on some error message after the null coalescing operator

Reply:
On second thought that seems a little extra, maybe just mark this with a comment for now? Especially since the other fields of `enodeb_state` are also optional for some reason

Reply:
Optional fields will help us with versioning and handling cases when NMS will still be compatible with older and newer APIs.